src/components/commerce/CommerceReservationDetails.js

```diff
@@ -58,6 +60,14 @@ class CommerceReservationDetails extends Component {
       commerceId: this.props.commerceId,
       reviewId: this.state.reservation.receivedReviewId
     });
+
+    if (!this.state.reservation.commerceId)
+      this.setState({
+        reservation: {
+          ...this.state.reservation,
+          commerceId: this.props.commerceId
```

Comment:
al pedo setar el commerceId aca, porque no usarlo directamente de this.props ??

Reply:
Es porque despues le paso el objeto` this.state.reservation` a una action mas abajo, y tiene que tener el `commerceId`

Reply:
sisi lo vi pero bue, capaz lo podias mergear ahi directo

Reply:
Va a ser mas viaje mepa